deployment/ansible/roles/cac-tripplanner.app/tasks/jslibs.yml

```diff
@@ -11,24 +11,29 @@
        state=present
 
 - name: Install NPM dependencies
-  npm: path="{{ gulp_dir }}" state=present
+  npm: path="{{ root_src_dir }}" state=present
```

Comment:
We switched to using `npm install` directly in NYC because of the discussion here: https://github.com/azavea/nyc-trees/pull/124#issuecomment-66536399